rwengine/src/render/GameRenderer.hpp

```diff
@@ -140,8 +140,9 @@ class GameRenderer {
     void setupRender();
     void renderPostProcess();
 
-    Renderer* getRenderer() {
-        return renderer.get();
+    Renderer*& getRenderer() {
```

Comment:
A reference to a pointer does not make sense here.
You're now basically returning a pointer to pointers in disguise.